### DISCUSSION TITLE
Troubleshooting installing Elementary 7

### DISCUSSION THREAD

**Discussion by Mirabulle** [elementary/installer#709] [Q&A]
Is it possible to install elementary on SD card ?

I have created 3 partitions: one /home, one /boot/efi and one for swap.
But installer leave me always same error:
“Extracting error: archive extraction failed with status: exit status 1”

I created 2 USB flash drive with 2 different ISO files but I have always same error.

Is it possible installer failed because I would like install elementary on SD card ? 

Could you please help me?

**Comment by lenemter** [comment]
What's the capacity of SD card and how much RAM do you have?

**Reply by Mirabulle** [reply]
120 G on my SD card and 4G for RAM

**Reply by Mirabulle** [reply]
I've allocated 30G for home, 1G for boot and 6G for swap

**Reply by lenemter** [reply]
@Mirabulle I think you need a main partition "/" as well.

**Reply by Mirabulle** [reply]
I understand but installer don't let me this choice.
I can just create /home, /boot and swap.

**Reply by Mirabulle** [reply]
Is there a way to force it to do a /

**Reply by lenemter** [reply]
I don't know. To be honest, I never even partitioned /home and swap partitions. I usually just go with / and /boot/efi. Can you try it?

**Reply by Mirabulle** [reply]
I do it
Always same error 
I read more details on log
It failed to extract /cdrom
I don't have cdrom reader on my laptop 
Is there reason it failed ?
It can't find cdrom so it crashed ???

**Reply by lenemter** [reply]
I looked at distinst bug tracker while searching for similar issues and found out that /cdrom might actually refer to the flash drive you're installing from. How are you flashing your flash drive?

**Reply by Mirabulle** [reply]
One with recommendated etcher 
And one with rufus 
USB drive capacity is just 4 G
Maybe not efficient ?

**Reply by lenemter** [reply]
The extracted iso file is only 2.8 GB, so that shouldn't be an issue. I have no guesses here. Maybe someone else knows why it's happening. Sorry for this.

**Reply by Mirabulle** [reply]
Thank you for your help.
I learned something in all case

**Comment by mikydus** [comment]
I haven't been able to do much with Elementary Linux 7. In short, according to me it's a big spike!! He looks beautiful, but unfortunately, that's all he is in reality! I can't even open a file as root, your Linux Elementary 7 system doesn't allow me to do this simple operation. You wanted to secure everything, this is what you have: a system only good for children!! On this Linux Elementary 7 I can teach the kids to spin around the queue in peace!! With you in the lead!!

**Reply by lenemter** [reply]
Insults and non-constructive feedback is not welcome here. If you want to get **any** support, you need at least to be respectful to whom you are asking.
